autonomy/fsm/scaffold/scaffold_skill.py

```diff
@@ -155,20 +166,49 @@ def _update_models(  # pylint: disable=no-self-use
             "tendermint_dialogues", SkillComponentConfiguration("TendermintDialogues")
         )
 
+    @classmethod
+    def get_actual_abstract_round_abci_package_public_id(
+        cls, ctx: Context
+    ) -> Optional[PublicId]:
+        """Get abstract round abci pacakge id from the registry."""
+        package_manager = PackageManagerV1.from_dir(Path(ctx.registry_path))
+        packages = [
+            package_id.public_id
+            for package_id in package_manager.dev_packages.keys()
+            if package_id.author == VALORY_AUTHOR
+            and package_id.name == VALORY_ABSTRACT_ROUND_SKILL_NAME
+            and package_id.package_type == PackageType.SKILL
```

Comment:
Same as above, you can use the public ID constant for the abci skill from here https://github.com/valory-xyz/open-autonomy/blob/be27dc89034c6ab28cdd04baecbc67d8757a6496/autonomy/constants.py#L60

Reply:
nice point!